src/Driver/Compiler.php

```diff
@@ -521,4 +516,24 @@ protected function optional(string $prefix, string $expression, string $postfix
 
         return $prefix . $expression . $postfix;
     }
+
+    private function compileArrayCondition(array $values, QueryParameters $params, Quoter $q, bool $in): string
+    {
+        $operator = $in ? 'IN' : 'NOT IN';
+
+        $placeholders = $simpleParams = [];
+        foreach ($values as $value) {
+            if ($value instanceof FragmentInterface) {
+                $placeholders[] = $this->fragment($params, $q, $value);
+            } else {
+                $placeholders[] = '?';
+                $simpleParams[] = $value;
+            }
+        }
+        if ($simpleParams !== []) {
+            $params->push(new Parameter($simpleParams));
+        }
+
+        return \sprintf('%s (%s)', $operator, \implode(', ', $placeholders));
```

Comment:
```suggestion
        return \sprintf('%s(%s)', $operator, \implode(',', $placeholders));
```
to minimize the result query

Reply:
Thanks, changed